discovery-infra/tests/conftest.py

```diff
@@ -69,19 +83,23 @@ def get_api_client(offline_token=env_variables['offline_token'], **kwargs):
     return assisted_service_api.create_client(url, offline_token, **kwargs)
 
 
+@pytest.fixture(scope="session")
+def is_dev_env():
+    return not is_qe_env()
+
+
 @pytest.fixture(scope="session")
 def setup_node_controller():
     logging.info(f'--- SETUP --- node controller\n')
-    controller = nodeController(**env_variables)
-    controller.prepare_nodes()
-    yield controller
+    if is_qe_env():
```

Comment:
not sure i follower the logic here? I think all logic must move to nodes fixture and here just return the relevant module.
Why to have same if's in different files

logging.info(f'--- SETUP --- node controller\n')
yield nodeController

Reply:
Fixed